ci/pipeline-vps.yml

```diff
@@ -0,0 +1,81 @@
+resources:
```

Comment:
Why is this file in this PR? OK.. I guess make sure it's removed when resubmitting this since I merged previous PR